test/WebSites/RazorWebSite/Services/TestHeadTagHelperComponent.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace RazorWebSite
+{
+    public class TestHeadTagHelperComponent : TagHelperComponent
+    {
+        public TestHeadTagHelperComponent()
+        {
+        }
+
+        public override int Order => 1;
+
+        public override Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
+        {
+            if (context.TagName == "head")
+            {
+                var customAttribute = new TagHelperAttribute("inject");
+                context.AllAttributes.TryGetAttribute("inject", out customAttribute);
```

Comment:
If `customAttribute` is an `out` parameter, why set it in the previous line? It'll just get overwritten anyway.